docs/simulating_with_the_mvs.rst

```diff
@@ -3,114 +3,138 @@ Simulating with the MVS
-* For non-dispatchable sources, eg. the generation of a PV plant, you need to provide a specific time series (unit: kWh/kWp, etc.). For the latter, make sure that its values are between zero and one ([0, 1]).
+.. note::
+    The name (or label) of each assets needs to be unique and used coherently across the various csv files.
 
+.. note::
+    If the user used the CSV format to simulate a local energy system, the MVS will automatically create a Json file (:code:`mvs_csv_config.json`) from the provided input data.
+    The user could rename this file :code:`mvs_config.json` and use it as input for the simulation.
 
 Json file: mvs_config.json
 ##########################
 
-In combination especially with the planned graphical user interface for the MVS (EPA),
-a json file will be used to store and provide all input data necessary to understand relation.
-The Json file itself is created by the EPA, ie. there are no manual changes.
-You can use a specific Json input file if you want to test a simulation that has been made public online,
-one test simulation, or as a developer that has knowingly edited the Json file.
+The structure of the Json file matches the one described by the :code:`csv_elements` folder. The Json format is intended for easier exchange: via http requests for online services such as EPA for example
+
+Use of Json file is recommended for advanced users only.
+
+There can only be a single Json file in your input folder and it must be named :code:`mvs_config.json`.
+
+An example of a Json file structure is available from the `default scenario <https://github.com/rl-institut/blob/dev/tests/inputs/mvs_config.json>`__ of the MVS
 
-As this requires to adhere to quite specific formatting rules,
-this can really only be recommended for advanced users.
 
-There can only be a single Json file in your input folder.
-As some parameters in the Json file link to a time series provided as a CSV,
-the folder "time_series" should be present in your input folder, as clarified in the next section.
+.. _time_series_folder:
+
+Time series: time_series folder
+###############################
+Some parameters value in the CSV/Json files are filenames. Those filenames correspond to files which must be present in the folder :code:`time_series` in your input folder, formatted as CSV.
+As an example, if one asset listed in :ref:`energy production <production>` has :code:`generation_pv.csv` as value for the :ref:`file_name <filename-label>`. The file :code:`generation_pv.csv` containing a value of the pv generation for each timestep of the simulation should be present in the :code:`time_series` folder.
+
+.. note::
+    The time series describing a non-dispatchable demand or when a time series defines an otherwise scalar value of a parameter (eg. energy price), the time series can have any absolute values.
```

Comment:
```suggestion
    When a time series describes a non-dispatchable demand or an otherwise scalar value of a parameter (eg. energy price), the values of the time series can have any positive value.
```

Reply:
Thank you it is much clearer than before, I meant to tag you on this one (I had only wrapped it within `.. note::`